_component_design/chips.md

```diff
@@ -9,28 +9,28 @@ preview-image: preview-images/chip.svg
 resource: true
 status: stable
 last-modified: 2017-11-16
-helix-ui-css: false
-helix-ui-javascript: false
 ---
 
 {% include toc.html %}
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### When to use
+## When to use
```

Comment:
This comment is going to serve as the discussion point for the conversation of H3's to H2's across the 5 documents involved in the PR. Few things to note on this conversion:

- Converting all `H3` sections header to `H2` elements and removing the `H2` that encapsulated them causes the file to lose segmentation and grouping. This is a concern. Can you detail the rational behind making this change?

- From a stylistic standpoint `H2` elements should never been contained within column elements. Content within the column elements will be rendered. H2 stylistically are larger and are meant to placed outside the div / column (See below)

![image](https://user-images.githubusercontent.com/586682/36863333-0331be8e-1d4f-11e8-9164-c15e15debb7f.png)

@araiford @palindromebart See above.